### ISSUE TITLE
Add missing content

### ISSUE THREAD

**Issue by EnisMulic** [OpenSourceSarajevo/rjecnik#31]
Add text for the following sections

* O Projektu
* FAQ
* Privatnost
* Uslovi korištenja

**Comment by Dash10107** [comment]
hey , can i work on this issue ? 
I have setup the project locally successfully , I just wanted to ask whether you would like to add content in English or Rjecnik ?

**Comment by EnisMulic** [comment]
@Dash10107 of course, I've added the needed content to the sub issues so you can use that
